src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 80;
```

Comment:
The return value for less than 3 days is hard-coded to 80, which does not follow the rule of `$40` per day. Use a constant for the daily rate and multiply by the number of days.